spp_change_request/i18n/ar.po

```diff
@@ -619,11 +619,6 @@ msgid ""
 msgstr ""
 "خطأ في مراحل التحقق من الصحة. لا توجد مرحلة متاحة لتعيين هذا التحقق من الصحة."
 
-#. module: spp_change_request
```

Comment:
@gonzalesedwin1123 Will the translations for `spp_dms_extra_documents` be generated automatically, or should this section be kept and changed to match the change in data/dms.xml?

Reply:
It should be kept @kneckinator and later be updated to match the change in data/dms.xml